mbta/semaphore/custom-settings.xml

```diff
@@ -0,0 +1,36 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.2.0"
```

Comment:
Just curious - why did you need to create this file? Was it auto-generated by some tool or did you have to make it yourself?

Reply:
I had made it myself because I kept getting build errors (both here and also locally) because Maven didn't like insecure URLs (see "Blocked mirror for repositories" [in the error here for example](https://github.com/mbta/OpenTripPlanner/runs/2923312837?check_suite_focus=true)). I think that's a relatively new "feature" of Maven. Do these errors not happen to other folks? 🤔

Reply:
Huh, weird, I had never run into that before.

Reply:
I wonder if there's some other configuration I'm missing.